app/templates/components/preprint-form-authors.hbs

```diff
@@ -1,56 +1,142 @@
-            {{input class="form-control" value=newContributorId}}
+<div class="row form">
+    <div class="col-xs-12 col-sm-6 col-division-right">
+        <div class="row">
+            <div class="col-xs-11">
+                <form {{action 'findContributors' 1 on='submit'}}>
+                    <div class="input-group search-box">
+                        <input type="text" class="form-control input-lg searchQuery" onchange={{action "updateQuery" value="target.value"}} placeholder="Search by name">
+                        <span class="input-group-btn">
+                            <button class="btn btn-default btn-lg" type="button" {{action 'findContributors' 1}} >
+                                <i class="glyphicon glyphicon-search"></i>
+                            </button>
+                        </span>
+                    </div>
+                </form>
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-xs-11">
+                {{#if (not (eq addState 'emptyView'))}}
+                    {{#if (eq addState 'searchView')}}
+                        <div class="unregisteredUsers">
+                            <p> Can't find the user you're looking for? </p>
+                            <button class="btn btn-primary btn-small" {{action "unregisteredView"}}> Add an unregistered user </button>
+                        </div>
+                        <h3> Results </h3>
+                        {{#if newSearchResults}}
+                            <table class="table author-table">
+                                {{#each newSearchResults as |result|}}
+                                    <tr>
+                                        <td>
+                                            <img class="m-l-xs" src={{result.links.profile_image}} height=40 width=40>
+                                            <a href={{result.links.html}}> {{result.fullName}} </a>
+
+                                        </td>
+                                        <td> <button class="btn btn-success btn-small" {{action "addContributor" result}}> Add </button> </td>
+                                    </tr>
+                                {{/each}}
+                            </table>
+                        {{else}}
+                            No results found.
+                        {{/if}}
+                        {{#if (gt pages 1)}}
+                            {{osf-paginator totalSearchResults=totalSearchResults maxPages=5 fetchResults=findContributors query=query}}
+                        {{/if}}
+                    {{/if}}
+                    {{#if (eq addState 'unregisteredView')}}
+                        <h3> Add Unregistered User </h3>
+                        <div class="user-validation">
+                            {{unregistered-contributor-form emptyView=(action 'emptyView') addUnregisteredContributor=(action 'addUnregisteredContributor')}}
```

Comment:
I had just named it the empty view.  There are three possible views here.  This just switched it to the empty view. You're right - it's poorly named.